OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -782,6 +774,131 @@ void testFunctionBasedBushingForce()
 
     ASSERT(*copyOfSpring == spring);
 }
+
+void testExpressionBasedBushingForce()
+{
+    using namespace SimTK;
+    
+    double mass = 1;
+    double stiffness = 10;
+    double start_h = 0.5;
+    double ball_radius = 0.25;
+    
+    double omega = sqrt(stiffness/mass);
+    
+    double dh = mass*gravity_vec(1)/stiffness;
+    
+    // Setup OpenSim model
+    Model *osimModel = new Model;
+    osimModel->setName("ExpressionBasedBushingTest");
+    osimModel->setGravity(gravity_vec);
+
+    // Create ball body and attach it to ground
+    // with a vertical slider
+
+    const Ground& ground = osimModel->getGround();
+
+    OpenSim::Body ball("ball", mass, Vec3(0), mass*SimTK::Inertia::sphere(0.1));
+    ball.addMeshGeometry("sphere.vtp");
+    ball.scale(Vec3(ball_radius), false);
+    
+    SliderJoint sliderY("", ground, Vec3(0), Vec3(0,0,Pi/2), ball, Vec3(0), Vec3(0,0,Pi/2));
+    
+    double positionRange[2] = {-10, 10};
+    // Rename coordinates for a slider joint
+    CoordinateSet &slider_coords = sliderY.upd_CoordinateSet();
+    slider_coords[0].setName("ball_h");
+    slider_coords[0].setRange(positionRange);
+    slider_coords[0].setMotionType(Coordinate::Translational);
```

Comment:
Why is this line necessary? Aren't coordinates associated with `SliderJoint`s necessarily translational?